src/lib.rs

```diff
@@ -236,6 +239,15 @@ pub fn enclave_console(enclave_cid: u64) -> NitroCliResult<()> {
     )
     .map_err(|e| e.add_subaction("Connect to enclave console".to_string()))?;
     println!("Successfully connected to the console.");
+
+    // Start the timeout only if the option was provided
+    if disconnect_timeout != None {
+        thread::spawn(move || {
+            thread::sleep(Duration::from_secs(disconnect_timeout.unwrap()));
+            process::exit(0);
```

Comment:
Neah, this not the right way to terminate the console, we should have a read_to_with_timeout that exits when the timeout expires.